### PR TITLE
userns KEP followup

### PR DESCRIPTION
**Pull request by giuseppe**
address some comments we got on PR#3065

### CODE REVIEW

**Comment by giuseppe** [comment]
@rata we can discuss them further before pushing to upstream PR

**Review by rata — COMMENTED** [review]
@giuseppe thanks! Sorry again I missed the notification :(

Added some comments. Also, IIUC the typos mrunal commented on are missing

File: keps/sig-node/127-user-namespaces/README.md [hunk 1]
```diff
@@ -72,8 +67,7 @@ Items marked with (R) are required *prior to targeting to a milestone / release*
 ## Summary
 
 This KEP adds a new `hostUsers` field  to `pod.Spec` to allow to enable/disable
-using user namespaces for pods. Furthermore, it allows increased pod to pod
-isolation by means of `pod.spec.securityContext.userns.pod2podIsolation` field.
+using user namespaces for pods.
 
 It allows users to place pods in different user namespaces increasing the
```
Comment (rata) [inline keps/sig-node/127-user-namespaces/README.md:72]: I think this paragraph should go now (referred to phase 3)

**Review by rata — COMMENTED** [review]
One silly comment. I think we still need to fix the typos mrunal found. Will have a look at the complete doc now, not just the diff, just in case

**Pull request merged**